### DISCUSSION TITLE
NatsMQ over RocksMQ

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [milvus-io/milvus#29761] [Q&A and General discussion] [answered]
I am using Milvus 2.2.6 and facing the issue of increasing size in `rdb_data` folder on each search. I searched over the Repo and found out that the issue lies with RocksMQ. I am considering using different MQ engine. Is NatsMQ perofrming better than RocksMQ? Or is it raising the same issue?

P.S.: I cannot change my Milvus version due to other factors, so I request anyone reading this to help me out with the specified version.

**Comment by xiaofan-luan** [comment]
RocksMQ is still recommended. 

2.2.6 seems to be too old, any reason we have to keep using the old version?
There are many stability fixes that you will need to backport

**Reply by ArijitSinghEDA** [reply]
I cannot disclose the reason, but I can ask to upgrade to the newer version.
But the question remains, is the folder `rdb_data` still increasing in size after every search?

**Comment by xiaofan-luan** [comment, marked as answer]
there are many fixes related, I think you will need to backport and also tune the log purge time.
Unfortunately 2.2.6 is too old and 2.2.16 and 2.3 branch is actively maintained.

**Reply by xiaofan-luan** [reply]
search for rocksmq && rdb && disk and you should get the related issue
